src/PooledArrays.jl

```diff
@@ -307,7 +309,7 @@ Base.convert(::Type{Array}, pa::PooledArray{T, R, N}) where {T, R, N} = convert(
 function Base.getindex(pa::PooledArray, I::Integer...)
```

Comment:
maybe allow to propagate inbounds?

Reply:
Let's do that elsewhere? There are so many little things to improve in this package...

Reply:
OK, I hadn't realized it was related to the changes in this PR. I've added `Base.@propagate_inbounds` and an `@inbounds` annotation (since we know references are correct).

Reply:
Actually that's already the default.

Reply:
OK